runelite-client/src/main/java/net/runelite/client/plugins/runecraft/RunecraftPlugin.java

```diff
@@ -255,4 +290,42 @@ public void onGameTick(GameTick event)
 			darkMage = result.length >= 1 ? result[0] : null;
 		}
 	}
+
+	@Subscribe
+	public void onGameObjectSpawned(GameObjectSpawned event)
+	{
+		GameObject obj = event.getGameObject();
+		int id = obj.getId();
+
+		if (id == NullObjectID.NULL_10796)
```

Comment:
You could define new constants with better names for these objects in this file. Also maybe switch().